drivers/util.py

```diff
@@ -624,7 +624,7 @@ def zeroOut(path, fromByte, bytes):
 
     blocks = bytes / blockSize
```

Comment:
Does this code work properly in Python 3 since blocks may be a float which will throw off the calculations below?

```
$ python2 -c 'print 5000/4096'
1

$ python3 -c 'print(5000/4096)'
1.220703125
```

Use `//` if you want the old Python 2 behaviour.

Reply:
good spot, I'll catch that up in the Python3 migration changes.